packages/process/src/node/terminal-process.ts

```diff
@@ -47,11 +47,10 @@ export class TerminalProcess extends Process {
 
     constructor(
         @inject(TerminalProcessOptions) options: TerminalProcessOptions,
-        @inject(ProcessManager) processManager: ProcessManager,
-        @inject(ILogger) logger: ILogger) {
+        @inject(ProcessManager) processManager: ProcessManager) {
         super(processManager, logger);
 
-        this.logger.debug(`Starting terminal process: ${options.command},`
+        logger.info(`Starting terminal process: ${options.command},`
```

Comment:
We should use injection as much as possible to keep things simple. I don't think that accessing stuff outside of inversify context should be encouraged.

Reply:
This is juste meant as a proof of concept test it's not proposed to be merged